initial ci tests for cma-aws

oneilcin opened the pull request:
These reflect the Swagger UI formats, but I could not create a cluster due to AWS IAM privilages not right.  (need privs corrected for creating the cloud formation instances)

zachpuck commented:
While completing a test the "delete" call never finishes. it stays at `Cluster DELETE is NOT COMPLETE` until `Timed out waiting for DELETE to finish` even though the cluster did actually delete and is showing a `does not exist` in the swagger ui

oneilcin commented:
I just added the fix for that.  The get-cluster.sh needs all fields (creds, and region) to return the right status.  I had removed it because I thought it was working without the creds.

davidewatson commented on test/e2e/delete-cluster.sh:
```diff
@@ -0,0 +1,51 @@
+set -o nounset
+set -o pipefail
+
+rawurlencode() {
```
🤕
